server/static/bindata_assetfs.go

```diff
@@ -87,7 +87,7 @@ func (fi bindataFileInfo) Sys() interface{} {
21\xda\xf3\xb6\xcd\xb1\x6d\xd1\xdf\xaa\xf6\x6f\x15\xa4\xf7\x0c\xb3\xc9\x3b\x26\xd6\x6e\x00\x5a\x9a\x06\xde\x13\x4e\xc3\x38\xea\x21\xa2\xfd\x9f\x4c\x4c\x6e\xa5\x31\xf2\x1e\x61\x06\xa9\xbc\xc0\xb9\x50\x1c\x75\x80\x83\x58\xdf\xf7\x4a\xc5\x34\xc1\xfa\xb8\x3f\x28\xd9\x53\x19\xef\xc1\x38\xcb\x30\x87\x09\xa3\xdc\xe9\x8e\xec\xb6\x22\xfd\x51\x32\xc4\xcd\xda\xcf\x92\x36\xfe\x35\xd5\x7e\xa5\x33\x1f\xb1\x83\x06\x2c\x30\x29\x4d\xf4\x41\xbd\xcd\xe5\x87\x8b\xee\xac\x1a\x95\x26\x75\xf3\xc3\x9c\x6d\x2a\x44\xd2\x6f\xc7\xbc\xee\x07\x6e\xb2\x25\x2a\xa9\x2a\x88\x2c\xa3\x5c\xbb\x2a\x3f\x82\x1f\x2c\x47\x2b\x92\xce\x45\x4b\x34\xe7\x55\xac\x34\x51\x7b\xd3\x82\x44\xa1\xaf\x7a\xef\xbc\x6f\x44\x68\x8c\x9e\x29\xbe\x84\xad\x20\xe5\xdd\xe8\xbc\x09\xd3\x7c\x8d\x4a\x91\x1d\x2e\x5b\xb2\x4f\x53\x88\x12\xbf\xcf\xde\xe1\xc3\xfa\xcb\xe3\xe3\xa7\xb4\x3f\x48\xf7\x1e\x05\xa2\xaf\xf6\x67\x20\x33\x42\x29\x85\x7c\x4d\xf2\x97\x4f\xb7\x8f\x37\x8f\x03\xc9\xdd\xa3\x19\xc9\xce\x2f\xfd\x7d\x66\x7a\xe7\xd2\x6c\x32\x5d\x8e\x37\x0a\xe9\x76\xb2\x9d\x08\x45\x6a\x7e\x5f\xd4\x05\x56\x77\xea\x34\xf0\x4c\x98\x48\xf1\x62\x5e\x16\x07\xa9\x8c\xc2\x25\x56\xe4\xc0\x74\x58\x22\xfc\x7b\x8f\xf0\x37\x51\x12\x06\x1f\x3b\x0b\x5c\x98\x46\x32\xae\xed\xe3\xd9\x99\xe9\xfa\x12\xfe\xa0\x65\x89\x1c\xf2\x53\x90\xe8\x9a\xcf\x69\x17\x6e\x88\xff\xa5\xc9\x09\x28\x07\xbb\x0b\x0b\x84\xed\xc0\x97\x3a\x1a\xaa\xc1\x76\xcc\x4d\x2b\xc9\x87\xbf\x33\x9b\xdb\x37\x9b\x89\x58\x14\xae\xb9\x16\x95\x6d\xbd\x73\x71\x0c\x1c\x83\x56\xb4\xdf\xa3\x0e\xa6\xb8\xeb\x4b\xf8\x7a\x60\xcc\xf7\xec\x9e\x75\x30\xa0\xb7\x24\xee\x69\xa0\x31\x58\x54\x31\xf1\xe2\x97\xd0\x86\xea\x0b\x37\x33\x39\xa8\x42\x0a\xc6\x80\x56\xc0\x11\x4b\x2c\x03\xc7\x4c\xce\xed\x72\xd7\x21\x5e\xb8\x53\x08\x63\x86\xc8\xc5\xd3\xab\x6c\xa1\xb3\x4c\xe2\x5b\xc7\xfb\x3b\x33\x8c\x2f\xd7\x20\x4c\xc9\xd1\x27\xc3\xed\xbc\xea\x3c\xfa\x87\x5b\xcc\xb7\xbe\x8c\x7a\x9b\xfa\x51\x9d\x0d\x41\x39\x9f\x61\xf6\x67\x50\xc2\xed\x82\xeb\x61\x70\xda\xef\xa2\x3c\xf5\xce\x72\xcb\xe6\xf6\x90\xd4\x9e\x72\x67\x97\xcd\x1d\xcf\xa3\x07\xea\x8e\xad\x0f\xdd\xcb\x8d\xc4\x8c\x92\x15\x9a\x7f\xc3\x16\x25\x7c\x55\x30\x6d\x62\x66\x06\x92\xed\x76\x3b\x88\x95\x5e\xe3\x7c\xf4\x7d\x7e\xe6\x8b\xe7\xd6\x56\xe3\x81\x4b\xd6\x17\x57\x6e\x04\x30\x1d\xcb\xf8\x65\x7a\x7f\x31\x9c\x5a\x08\xa7\xb5\x8d\x61\xbb\x65\xce\xc0\x7d\x46\x2d\x9a\x25\xba\xf2\x10\x9a\x9a\x24\xbe\x55\x8e\xea\x2d\x29\x4b\xdc\xde\x6d\x9f\xcb\x12\x3e\x07\x12\xbb\x54\x0a\xc7\x3e\xe1\xa9\x92\xa4\x46\xd5\x13\x19\x36\x67\x52\xd4\xf0\xdd\xa6\x56\x74\xe3\x52\xd2\x47\x5f\x06\xc9\x19\x7c\xe2\x79\x8a\xa4\xf7\x32\xb5\x0b\xad\xdf\x7e\x5a\xf4\x5b\x92\xe3\xff\xa8\xb6\x22\xaa\x43\x51\xa0\x52\x3f\xb8\x6e\x1f\x76\x7f\xa9\xe9\xd9\xe6\x97\xe1\xab\xd8\x36\x0c\xd3\xaf\x59\xfe\x0c\xf1\x70\x86\xd5\x2a\x56\x35\x61\x6c\xfa\xa5\x55\xe2\xbe\x4b\x30\x5e\xfb\x2a\x84\xe9\x85\xfc\x56\x40\x59\x04\xfc\xac\x19\xe1\x9a\x2a\x78\x46\xa9\xbc\x33\x2b\x47\xf7\x33\x3a\x86\x15\xcf\xfc\x8c\xd7\x6d\x9f\x56\x00\xb2\x37\x22\x8e\xbe\x9e\x18\x35\x2f\xa9\xaf\xfb\xff\x0b\x00\x00\xff\xff\xc7\xb2\xfa\x8e\x53\x1c\x00\x00")
```

Comment:
Could you describe this change?

Reply:
This is the result of running `make dist` to reflect the changes made in [custom.css](server/static/css/custom.css)

Reply:
Thank you.

Please add that as a comment

```suggestion
// This line changes using `make dist` to reflect the changes made in custom.css
var _staticCssCustomCss = []byte("\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\xff\xac\x59\xeb\x6e\xdb\xca\xf1\xff\xae\xa7\x98\xff\x09\x0e\x10\xfb\x6f\xd2\xa4\x6c\x39\x32\x0d\x14\x27\xc7\x48\x70\x3e\xb4\x45\xd1\xf6\x05\x96\xe4\x50\xda\x7a\xb9\xcb\xee\xae\x6c\x29\x81\xde\xbd\xd8\x1b\xef\xb4\xe3\xe0\x38\x88\x61\x91\x33\xb3\xb3\x73\xf9\xcd\x45\x71\x21\xb8\x26\x94\xa3\x84\xef\x2b\x80\x9a\x1c\xa3\x17\x5a\xea\x7d\x06\x69\x92\x24\xcd\xf1\x01\xce\xab\x78\x8f\xa4\x6c\x09\xe4\x8e\xf2\x48\x8b\x26\x83\x75\xbc\x79\xde\x3f\xac\x00\x34\x1e\x75\x44\x18\xdd\xf1\x0c\x0a\xe4\x1a\xa5\x61\xa3\xf5\x2e\xde\xa3\x14\x96\x6f\x8f\x74\xb7\xd7\x19\xac\x13\xc7\x62\x0e\x0a\xcf\xd2\xb5\x39\x68\x05\xe0\x4f\x26\x07\x2d\x1e\x86\xca\xac\x6f\xd3\xe6\xb8\xf2\xaa\x50\xbe\x8b\x2a\xc1\x75\xa4\xe8\x37\xb4\xd2\xdb\x4f\x19\xa4\xf1\x5a\x62\x6d\xd8\x0b\xc1\x84\xcc\xe0\xc3\xfd\xfd\xbd\xf9\xc8\x50\x6b\x94\x91\x6a\x48\x41\xf9\x2e\x03\x2e\x64\x4d\x98\xbd\x5f\x21\x4a\x8c\xf0\x48\xea\x86\xe1\xe4\x96\x69\xbc\xf1\x02\xfd\xd3\x5c\x68\x2d\xea\x0c\x92\x09\x6f\x94\x8b\xf2\x64\x05\xbc\xec\xa9\x46\x7b\x16\x66\xd0\x48\xb4\xb7\x13\xb2\x8c\x5e\x24\x69\x32\xc8\x25\x92\xa7\xc8\x3c\x30\x22\x38\x79\xce\x89\x33\x6f\x49\x55\xc3\xc8\xc9\x68\xc7\xd1\xcb\xe7\x1a\xb9\x8e\x34\xc9\x19\x46\xfe\xfa\x93\x4b\xdf\x36\xc7\x87\xf3\x4a\xb7\xe7\xf7\xdf\xad\xed\xbb\x98\xd4\x34\x2a\x51\x13\xca\x94\x25\xf1\x6e\xd3\x92\x70\x55\x09\x59\x67\x70\x68\x1a\x94\x05\x51\x56\xdb\x05\xa3\x4e\xed\xd8\x7b\x65\x39\x5e\xbc\x57\xef\x92\xe4\xa1\x3b\xc6\x47\x07\xc3\x4a\x3f\xac\x82\x36\x44\x93\x05\x55\xcc\xf5\xe1\xff\x68\xdd\x08\xa9\x09\xd7\x7f\x8a\x42\xa9\x51\xe8\xbc\x8a\x29\x57\x9a\xf0\x02\x23\x4e\x6a\x8c\x34\xe9\x59\xb3\x22\x35\x65\xa7\x0c\x6a\xc1\x85\x75\xde\x55\xf7\xe7\xc3\x38\xce\x6e\xed\x31\xe7\x55\xdc\x30\x52\xe0\x5e\xb0\x90\x23\xef\x17\x35\x0a\xd9\x9d\xc4\x53\x47\xa3\x4f\x0c\x33\xa0\x9a\x30\x5a\x2c\x64\x9b\xd1\x82\x09\x97\x19\xb4\xde\xcd\x84\xd2\x32\x9f\xc4\x1d\x15\x3c\x52\xc8\xb0\xd0\x96\x73\x8e\x10\xa0\x62\x82\xe8\x19\xcf\x9c\x57\x1f\xbc\x08\x46\x95\x9e\xa4\xcf\xd6\xa4\xf6\x79\x15\xff\xf7\x80\x07\x8c\xa8\xc6\xba\x4f\x62\xe2\xc1\x80\xc2\x84\x68\x10\xaa\xed\x55\x28\x67\x94\xa3\xcb\x85\x87\x79\x29\x00\x0d\x29\x4b\x1b\x08\x49\xfb\x28\x40\xc1\x66\xb3\x99\xd8\xfe\xc6\xdb\xde\x4a\x0e\x90\x74\xb3\x75\x8c\x93\xd8\x4a\x3d\xf5\x20\xc1\xb9\x30\x69\x6d\x1e\xe7\x42\x96\x28\x23\x49\x4a\x7a\x50\x19\xdc\x3a\x29\xee\x69\x06\x69\x73\x04\x25\x18\x2d\xe1\x43\x9e\xe7\xee\xcd\xd1\x28\x62\x65\x7b\xde\x5c\x58\x5b\xac\xae\x2f\xe1\xaf\x44\xee\x50\x82\xde\x13\x0e\xcd\xde\x18\xfe\xf2\x7a\xf5\x5b\x8d\x25\x25\xf0\xb1\xa6\x3c\xa0\xe3\x66\x93\x34\xc7\x0b\x6b\xaa\xf8\x99\xb0\x03\x46\x8d\x14\x4d\x48\xf3\xbe\x37\xee\xdb\xd4\x18\x81\xd9\x27\xf3\x02\xce\x9d\x84\x10\x46\x13\xca\xb4\xa3\x0c\xb0\xb7\x93\xb4\x84\xb8\x10\xec\x50\xf3\xab\xa5\x17\x6a\x41\x9c\x83\x57\x27\xb0\x14\x85\x8a\x14\x16\x9a\x0a\xee\xc9\x5b\x6f\xde\x49\xac\x1d\xe6\xf6\x0e\x50\xc8\xcb\xe8\x24\x0e\x52\x21\xab\xa2\x42\x34\x27\xcf\xe6\xa3\x55\x1a\x77\x3e\x4c\xcc\x60\x31\x71\x24\xa9\x90\x88\x5c\xed\x85\xb6\x18\xdd\xf8\x5c\x06\x68\x84\xa2\x46\x9f\x0c\x48\xae\x04\x3b\x68\x0f\x8f\xde\xf6\xb7\xdb\x5f\xdd\xe7\x7d\x07\x33\xbf\x06\x60\x32\x61\x99\x84\xe3\xbb\x82\x17\xc0\x7d\xea\x65\x1b\xd9\x7a\xc1\xcd\x9f\x3a\x37\x5f\x5f\xc2\xdf\x5d\xd5\xb8\xbc\x36\xb7\xf0\x25\xe4\xff\x67\x4d\xe8\x03\x4b\x8b\x26\x48\x0a\x66\x74\x6c\x57\xdd\x9f\x2e\xa2\xc3\xd5\xdb\xbc\xcb\x99\x28\x9e\x06\xd7\xee\x6e\x19\x2e\x75\xd7\xd6\x49\x80\x9c\x14\x4f\x3b\x29\x0e\xbc\xcc\xe0\x43\x55\x55\xee\xe9\xb7\x88\xf2\x12\x8f\x19\xb8\x82\xdc\x57\x6c\x90\x1b\x88\x38\x78\xdf\x86\xca\x90\xa4\x7f\x85\x05\xbd\x83\xa1\x3b\x0b\xfd\x05\x46\x1d\xcf\xf0\x46\x03\x99\x2d\x9e\x19\x70\x50\x2d\x12\x07\x3c\x9d\x6f\x08\x3a\xf6\x16\xeb\xfa\x31\x24\x91\x11\x4d\x9f\x43\x89\x75\x51\xea\xaa\xe2\xdb\x12\x19\xe5\x4f\xef\xaf\xdd\xa9\x03\xa1\xa5\xf2\xfc\x4a\xf9\xb4\x87\x94\x58\x08\x49\x9c\xf6\xdd\xd5\x07\x68\xd9\x77\x7d\x80\xda\xf5\x7a\x3d\x55\x3e\x26\x85\xb9\xbc\xbf\x43\x20\xbd\xb9\x79\xbc\xf9\x1a\xae\xba\x27\x2a\x2a\x45\xf1\x84\x65\xc4\xc9\x33\xf4\x9b\xa3\xbe\x1d\x2b\x7a\xc4\xd2\x6b\x69\xc2\x27\x19\xa6\xdc\x6b\xb2\x5e\x8d\x71\xcb\x78\x7d\x09\xff\xc4\x48\x3c\xa3\x94\xd4\xb6\x5b\x7a\x8f\x2e\x4c\x6c\x94\x40\x89\x05\x23\xce\x26\xa0\x05\xd4\x44\x17\x7b\xb0\xcd\xa8\xa8\xe0\x57\xc8\x89\xc2\x12\xba\x30\x73\x29\xba\x70\xb1\xe5\x80\xdc\xba\x78\x74\xfa\xfc\x43\x34\x46\x1f\x2f\xab\x71\x9f\x62\xd1\x20\x5f\xcc\xd5\x73\x8f\x72\x36\x31\x5e\x85\xb7\x39\xab\xbe\x92\xdb\x33\x25\x6e\x9c\xc6\xa3\xa2\xe8\x8f\xb8\x5f\x0f\xd0\x32\xda\x04\x5c\x89\x5e\x30\x7f\xa2\x3a\xaa\x28\xd3\x46\x72\x29\x45\x13\xa9\x3d\x29\xc5\xcb\xc7\x04\x12\xb8\x6b\x8e\x20\x77\x39\xf9\x98\x5c\x99\x7f\x71\x7a\x71\xe1\x18\x0d\x6f\x2d\xbe\xfd\x14\xa3\xfd\x79\x0f\xe3\xc0\xca\x36\xe5\xb3\x8a\x4a\xa5\xa3\x62\x4f\x59\xd9\xbd\x31\xd1\x9f\x91\x4a\xa3\x43\xdb\x1f\x64\xc8\xb1\x12\x12\x5b\x18\xf7\x30\x38\x2a\x30\xad\xc5\x82\x0f\x9c\xfd\x2d\x38\x34\x44\x62\x68\x9f\xfd\x24\x91\xc1\x2f\xf0\xcb\x10\xba\x93\x01\xb4\x27\xaf\x07\x46\x23\xa8\x69\x09\x23\x7c\x46\xae\xd5\x00\x63\xdf\x63\x86\x61\x6d\xf2\x50\x60\xad\xbb\xde\x6c\xae\xa0\xfb\x95\x5c\xcc\x14\x83\xc0\x30\x0e\xc0\x6e\x78\x0d\x41\x36\x68\x11\xa3\x34\x69\x0b\xff\xcf\xb9\x60\xaa\xed\xcd\xf6\x0a\xba\x5f\xaf\x6b\x3b\x4e\x89\xa0\x6d\xba\xa0\x6d\x3a\xd1\xb6\x57\x93\xba\x3e\xb7\xcf\xdb\x25\xed\xb8\x64\x4d\xaf\xfd\xa6\xa0\xf1\xd9\x6d\xf9\x59\x2a\x68\x7d\xf4\x5f\x6c\x21\xda\xf3\xb6\xcd\xb1\x6d\xd1\xdf\xaa\xf6\x6f\x15\xa4\xf7\x0c\xb3\xc9\x3b\x26\xd6\x6e\x00\x5a\x9a\x06\xde\x13\x4e\xc3\x38\xea\x21\xa2\xfd\x9f\x4c\x4c\x6e\xa5\x31\xf2\x1e\x61\x06\xa9\xbc\xc0\xb9\x50\x1c\x75\x80\x83\x58\xdf\xf7\x4a\xc5\x34\xc1\xfa\xb8\x3f\x28\xd9\x53\x19\xef\xc1\x38\xcb\x30\x87\x09\xa3\xdc\xe9\x8e\xec\xb6\x22\xfd\x51\x32\xc4\xcd\xda\xcf\x92\x36\xfe\x35\xd5\x7e\xa5\x33\x1f\xb1\x83\x06\x2c\x30\x29\x4d\xf4\x41\xbd\xcd\xe5\x87\x8b\xee\xac\x1a\x95\x26\x75\xf3\xc3\x9c\x6d\x2a\x44\xd2\x6f\xc7\xbc\xee\x07\x6e\xb2\x25\x2a\xa9\x2a\x88\x2c\xa3\x5c\xbb\x2a\x3f\x82\x1f\x2c\x47\x2b\x92\xce\x45\x4b\x34\xe7\x55\xac\x34\x51\x7b\xd3\x82\x44\xa1\xaf\x7a\xef\xbc\x6f\x44\x68\x8c\x9e\x29\xbe\x84\xad\x20\xe5\xdd\xe8\xbc\x09\xd3\x7c\x8d\x4a\x91\x1d\x2e\x5b\xb2\x4f\x53\x88\x12\xbf\xcf\xde\xe1\xc3\xfa\xcb\xe3\xe3\xa7\xb4\x3f\x48\xf7\x1e\x05\xa2\xaf\xf6\x67\x20\x33\x42\x29\x85\x7c\x4d\xf2\x97\x4f\xb7\x8f\x37\x8f\x03\xc9\xdd\xa3\x19\xc9\xce\x2f\xfd\x7d\x66\x7a\xe7\xd2\x6c\x32\x5d\x8e\x37\x0a\xe9\x76\xb2\x9d\x08\x45\x6a\x7e\x5f\xd4\x05\x56\x77\xea\x34\xf0\x4c\x98\x48\xf1\x62\x5e\x16\x07\xa9\x8c\xc2\x25\x56\xe4\xc0\x74\x58\x22\xfc\x7b\x8f\xf0\x37\x51\x12\x06\x1f\x3b\x0b\x5c\x98\x46\x32\xae\xed\xe3\xd9\x99\xe9\xfa\x12\xfe\xa0\x65\x89\x1c\xf2\x53\x90\xe8\x9a\xcf\x69\x17\x6e\x88\xff\xa5\xc9\x09\x28\x07\xbb\x0b\x0b\x84\xed\xc0\x97\x3a\x1a\xaa\xc1\x76\xcc\x4d\x2b\xc9\x87\xbf\x33\x9b\xdb\x37\x9b\x89\x58\x14\xae\xb9\x16\x95\x6d\xbd\x73\x71\x0c\x1c\x83\x56\xb4\xdf\xa3\x0e\xa6\xb8\xeb\x4b\xf8\x7a\x60\xcc\xf7\xec\x9e\x75\x30\xa0\xb7\x24\xee\x69\xa0\x31\x58\x54\x31\xf1\xe2\x97\xd0\x86\xea\x0b\x37\x33\x39\xa8\x42\x0a\xc6\x80\x56\xc0\x11\x4b\x2c\x03\xc7\x4c\xce\xed\x72\xd7\x21\x5e\xb8\x53\x08\x63\x86\xc8\xc5\xd3\xab\x6c\xa1\xb3\x4c\xe2\x5b\xc7\xfb\x3b\x33\x8c\x2f\xd7\x20\x4c\xc9\xd1\x27\xc3\xed\xbc\xea\x3c\xfa\x87\x5b\xcc\xb7\xbe\x8c\x7a\x9b\xfa\x51\x9d\x0d\x41\x39\x9f\x61\xf6\x67\x50\xc2\xed\x82\xeb\x61\x70\xda\xef\xa2\x3c\xf5\xce\x72\xcb\xe6\xf6\x90\xd4\x9e\x72\x67\x97\xcd\x1d\xcf\xa3\x07\xea\x8e\xad\x0f\xdd\xcb\x8d\xc4\x8c\x92\x15\x9a\x7f\xc3\x16\x25\x7c\x55\x30\x6d\x62\x66\x06\x92\xed\x76\x3b\x88\x95\x5e\xe3\x7c\xf4\x7d\x7e\xe6\x8b\xe7\xd6\x56\xe3\x81\x4b\xd6\x17\x57\x6e\x04\x30\x1d\xcb\xf8\x65\x7a\x7f\x31\x9c\x5a\x08\xa7\xb5\x8d\x61\xbb\x65\xce\xc0\x7d\x46\x2d\x9a\x25\xba\xf2\x10\x9a\x9a\x24\xbe\x55\x8e\xea\x2d\x29\x4b\xdc\xde\x6d\x9f\xcb\x12\x3e\x07\x12\xbb\x54\x0a\xc7\x3e\xe1\xa9\x92\xa4\x46\xd5\x13\x19\x36\x67\x52\xd4\xf0\xdd\xa6\x56\x74\xe3\x52\xd2\x47\x5f\x06\xc9\x19\x7c\xe2\x79\x8a\xa4\xf7\x32\xb5\x0b\xad\xdf\x7e\x5a\xf4\x5b\x92\xe3\xff\xa8\xb6\x22\xaa\x43\x51\xa0\x52\x3f\xb8\x6e\x1f\x76\x7f\xa9\xe9\xd9\xe6\x97\xe1\xab\xd8\x36\x0c\xd3\xaf\x59\xfe\x0c\xf1\x70\x86\xd5\x2a\x56\x35\x61\x6c\xfa\xa5\x55\xe2\xbe\x4b\x30\x5e\xfb\x2a\x84\xe9\x85\xfc\x56\x40\x59\x04\xfc\xac\x19\xe1\x9a\x2a\x78\x46\xa9\xbc\x33\x2b\x47\xf7\x33\x3a\x86\x15\xcf\xfc\x8c\xd7\x6d\x9f\x56\x00\xb2\x37\x22\x8e\xbe\x9e\x18\x35\x2f\xa9\xaf\xfb\xff\x0b\x00\x00\xff\xff\xc7\xb2\xfa\x8e\x53\x1c\x00\x00")
```

Reply:
Ugh the whole thing gets removed when the male target is run? Can the make target be modified to prevent removing the comment?

Reply:
I am actually not sure how to 😅 - but at least for this PR I will try to keep this comment.